scipy/stats/tests/test_distributions.py

```diff
@@ -2922,6 +2922,20 @@ def test_sf_isf(self):
         xx = stats.invgamma.isf(y, 1)
         assert_allclose(x, xx, rtol=1.0)
 
+    @pytest.mark.parametrize("a, ref",
+                            [(100000000.0, -26.21208257605721),
+                            (1e+100, -343.9688254159022)])
```

Comment:
```suggestion
                             [(100000000.0, -26.21208257605721),
                              (1e+100, -343.9688254159022)])
```

Sorry for the noise, the linter is dearly missed :(.

After this, I will approve.